Mathlib/Logic/Equiv/Basic.lean

```diff
@@ -407,15 +407,41 @@ def sigmaOptionEquivOfSome {α} (p : Option α → Type v) (h : p none → False
       exact rfl
   (sigmaSubtypeEquivOfSubset _ _ h').symm.trans (sigmaCongrLeft' (optionIsSomeEquiv α))
 
+section Sigma
+
+variable {ι : Type*} {π : ι → Type*} {p : ι → Prop}
+
+/-- The `Sigma` type indexed by a subtype can be canonically identified with a subtype of the
```

Comment:
below, you simply type `sigma`. (And I guess that entering ``Sigma`` suggests to enter ``Subtype``…)

Reply:
Both `Sigma` and "sigma" are used in this file, but never `Subtype`. I think I'll standardize to `Sigma` within this section and keep using "subtype".